server/routes/disasters.py

```diff
@@ -39,6 +43,10 @@ def disaster_dashboard_v0():
 @bp.route('/')
 @bp.route('/<path:place_dcid>', strict_slashes=False)
 def disaster_dashboard(place_dcid=DEFAULT_PLACE_DCID):
+  if place_dcid == "event":
+    # Access '/event' route instead
+    event_node()
+    return
```

Comment:
Sorry for the miscommunication - my preference would be for this to be in a separate route (/event/event_id rather than /disasters/event/event_id). Then we can avoid this, and the URL space is cleaner.

Reply:
Thanks for the clarification - done!